### PR TITLE
Basic vyper installation instructions

### PR DESCRIPTION
**Pull request by shellygr**

### CODE REVIEW

**Comment by urikirsh** [comment]
Going over the stale open doc PRs. @shellygr is this still up to date? If so, I'll review it

**Comment by shellygr** [comment]
Pretty vanilla, should be up to date

File: docs/user-guide/getting-started/install.md [hunk 1]
```diff
@@ -332,7 +332,18 @@ Step 4: Add the Solidity compiler (`solc`) executable's folder to your `PATH`
...
+
+[Vyper](https://github.com/vyperlang/vyper) is an EVM compatible Pythonic smart contract language.
+Since the Certora Prover operates on the bytecode, it can be applied to any source-level language
+that compiles to EVM bytecode.
+
+We recommend to install Vyper either from PyPi (i.e., `pip install vyper`) or to get a 
+binary executable for the desired version.
+
+
+Step 6 (for VS Code users): Install the Certora IDE Extension
```
Comment (urikirsh) [inline docs/user-guide/getting-started/install.md:346]: We no longer mention the vs code extension in our documentation as it is not maintained anymore

**Review by urikirsh — APPROVED** [review]

File: docs/user-guide/getting-started/install.md (hunk 1 above)
Comment (shellygr) [reply docs/user-guide/getting-started/install.md:346]: I think it's the lsp that should be mentioned

**Pull request merged**